src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -135,10 +135,12 @@ public abstract partial class HashAlgorithm : System.IDisposable, System.Securit
         public static System.Security.Cryptography.HashAlgorithmName SHA512 { get { throw null; } }
         public override bool Equals(object obj) { throw null; }
         public bool Equals(System.Security.Cryptography.HashAlgorithmName other) { throw null; }
+        public static System.Security.Cryptography.HashAlgorithmName FromOid(string oidValue) { throw null; }
```

Comment:
I used `dotnet msbuild /t:GenerateReferenceSource` but it made some other changes that I was not expecting, so I reverted those.

In particular, it removed `private readonly int _dummyPrimitive;` from `HashAlgorithmName`.